### PR TITLE
Nrma free2 go

### PR DESCRIPTION
**Pull request by tjharrop**
I wasn't going to let the week end without completing this!

Possible tech-debt: a value may be returned for a discount if only 1 part of the eligibility for it is entered (i.e. age when registering is provided but year of membership isn't)

Doesn't cause actual failures.

### CODE REVIEW

File: openfisca_nsw/tests/NRMA_free2go/act_and_citizen.yaml [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+        NRMA_free2go__membership_term: 3
+        birth: 2000-01-01
+        NRMA_free2go__is_NRMA_member: true
+        is_australian_citizen: true
+        NRMA_free2go__age_when_joining: 17
+    output:
+        NRMA_free2go__is_eligible: true
+        NRMA_free2go__free2go_discount: 0
+-
+    name: 'Eligible - 20 year old Australian Citizen in year 3 who is an ACT resident and NRMA member - receives 50% discount as they joined after turning 17'
```
Comment (UgoHarry) [inline openfisca_nsw/tests/NRMA_free2go/act_and_citizen.yaml]: Is this test criteria correct, compared to the previous one? (19yr, Year3, ACT, member, 0% discount)

File: openfisca_nsw/tests/NRMA_free2go/combinations.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        birth: 2003-01-01T00:00:00.000Z
+        NRMA_free2go__is_NRMA_member: true
+        is_australian_citizen: false
+        is_permanent_resident: true
+        NRMA_free2go__age_when_joining: 16
+    output:
+        NRMA_free2go__is_eligible: true
+        NRMA_free2go__free2go_discount: 100
+-
+    name: 'Is citizen but specifically not a perm resident'
```
Comment (UgoHarry) [inline openfisca_nsw/tests/NRMA_free2go/combinations.yaml]: Can person be a citizen without being a permanent resident?

File: openfisca_nsw/tests/NRMA_free2go/nsw_and_perm_resident.yaml [hunk 3]
```diff
@@ -0,0 +1,169 @@
+-
+    name: 'Eligible - 16 year old Australian Citizen in year 1 who is a NSW resident and NRMA member - receives 100% discount'
```
Comment (UgoHarry) [inline openfisca_nsw/tests/NRMA_free2go/nsw_and_perm_resident.yaml]: Tests are for permanent residents, not citizens as the names have suggested

File: openfisca_nsw/tests/NRMA_free2go/combinations.yaml (hunk 2 above)
Comment (tjharrop) [reply openfisca_nsw/tests/NRMA_free2go/combinations.yaml]: Yep! They're separate things, according to this it's an "or" not an "and"

File: openfisca_nsw/tests/NRMA_free2go/nsw_and_perm_resident.yaml (hunk 3 above)
Comment (tjharrop) [reply openfisca_nsw/tests/NRMA_free2go/nsw_and_perm_resident.yaml]: Fixed! New file should be in this PR now

Comment (tjharrop) [reply openfisca_nsw/tests/NRMA_free2go/nsw_and_perm_resident.yaml]: [f9cf959](https://github.com/digitalnsw/openfisca-nsw/pull/19/commits/f9cf959443f70eb8d0c1b80579073e8222611f0c)

File: openfisca_nsw/tests/NRMA_free2go/nsw_and_perm_resident.yaml [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+        NRMA_free2go__membership_term: 3
+        birth: 2000-01-01
+        NRMA_free2go__is_NRMA_member: true
+        is_permanent_resident: true
+        NRMA_free2go__age_when_joining: 17
+    output:
+        NRMA_free2go__is_eligible: true
+        NRMA_free2go__free2go_discount: 0
+-
+    name: 'Eligible - 20 year old Australian Citizen in year 3 who is a NSW resident and NRMA member - receives 50% discount as they joined after turning 17'
```
Comment (UgoHarry) [inline openfisca_nsw/tests/NRMA_free2go/nsw_and_perm_resident.yaml]: Name suggests 50% discount, output provides different discount of 0%

Comment (tjharrop) [reply openfisca_nsw/tests/NRMA_free2go/nsw_and_perm_resident.yaml]: Fiiiiiiiixed 🤘🤘🤘

File: openfisca_nsw/tests/NRMA_free2go/act_and_citizen.yaml (hunk 1 above)
Comment (tjharrop) [reply openfisca_nsw/tests/NRMA_free2go/act_and_citizen.yaml]: Fixed from the other comment

File: openfisca_nsw/tests/StEPS/does_not_meet_criteria.yaml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  output:
+    StEPS__child_meets_criteria: false
+- name: Not entitled due to age and school info not supplied
+  period: 2019-01
+  input:
+    birth: 2017-01-01
+    StEPS__already_screened: false
+  output:
+    StEPS__child_meets_criteria: false
+- name: Not child at 4 years old - already had StEPS screening
```
Comment (UgoHarry) [inline openfisca_nsw/tests/StEPS/does_not_meet_criteria.yaml]: Name is a bit confusing ("Not child at 4 years old")

**Pull request merged**